kent2cky/GeoSearch
# Chore  Update facebook share meta tags #28

**What does the PR do?**

Update facebook share meta tags 

**Description of tasks to be completed**

- Update facebook share meta tags to point to hosted GeoSearch app

**How should this be manually tested?**

- Visit the [GeoSearch](https://kent2cky.github.io/GeoSearch) page, search for any location. When the result page is displayed, click on the share button and check that the correct link and image are marked to be shared to facebook.

**Any background context you want to provide?**

nil

**What are the relevant pivotal tracker stories?**

#3
